app/modules/screen-sharing/controllers/AuthenticationCtrl.js

```diff
@@ -28,6 +28,15 @@ App.controllers.authenticationCtrl = (function ($, App) {
             desc = null;
         }
 
+        function clickExtension(e){
+            e.preventDefault();
+            chrome.webstore.install('https://chrome.google.com/webstore/detail/jlpojfookfonjolaeofpibngfpnnflne', function(){
+                console.log('Successfully installed Chrome Extension');
```

Comment:
Do we want to reload the page in this case since Respoke.js has no way of knowing whether it should check for the extension again? This would be a temporary workaround.